pennylane/io.py

```diff
@@ -122,11 +109,177 @@ def from_qiskit(quantum_circuit, measurements=None):
+        qc.measure_all()
+
+        @qml.qnode(dev)
+        def circuit():
+            # here measurements=None, so the measurements present in the QuantumCircuit are returned
+            measurements = qml.from_qiskit(qc)()
+            return [qml.expval(m) for m in measurements]
+
+    >>> circuit()
+    [tensor(1., requires_grad=True), tensor(0., requires_grad=True)]
+
+    .. note::
+
+        The returned measurements from the ``QuantumCircuit`` are in the measurement basis
+        and are readout values between 0 (ground) and 1 (excited). Such a measurement on the
+        `ith` wire does not match the result of `qml.expval(qml.PauliZ(i)`, which ranges from
+        1 (ground) to -1 (excited), indicating a position on the Bloch sphere.
+
+        I.e. we can alternatively translate the circuit and capture the same information in
+        a different format as:
+
+        >>> measurements = [qml.expval(qml.Z(0)), qml.expval(qml.Z(1))]
+        >>> circuit = qml.QNode(qml.from_qiskit(qc, measurements), dev)
+        >>> circuit()
+        [tensor(-1., requires_grad=True), tensor(1., requires_grad=True)]
+
+    See below for more information regarding how to translate more complex circuits
+    from Qiskit to PennyLane, including handling parameterized Qiskit circuits,
+    mid-circuit measurements, and classical control flows.
+
+    .. details::
+        :title: Parameterized Quantum Circuits
+
+        A Qiskit ``QuantumCircuit`` is parameterized if it contains ``Parameter`` or
+        ``ParameterVector`` references that need to be given defined values to evaluate
+        the circuit. These can be passed to the generated qfunc as keyword or positional
+        arguments. If we define a parameterized
+
+        .. code-block:: python
+
+            from qiskit.circuit import QuantumCircuit, Parameter
+
+            angle0 = Parameter("x")
+            angle1 = Parameter("y")
+
+            qc = QuantumCircuit(2, 2)
+            qc.rx(angle0, 0)
+            qc.ry(angle1, 1)
+            qc.cx(1, 0)
+
+        Then this circuit can be converted into a differentiable circuit in PennyLane and
+        executed:
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane import numpy as np
+
+            dev = qml.device("default.qubit")
+
+            @qml.qnode(dev)
+            def circuit(a0, a1):
+                # parameters are passed from the circuit function into the template as keyword arguments
+                # the keywords match the names given to the parameters in the code block above
+                qml.from_qiskit(qc)(x=a0, y=a1)
+                return qml.expval(qml.Z(0))
+
+        .. note::
+
+            The parameters can also be passed to the qfunc as positional arguments. In this case, the positions
+            match those from ``qc.parameters``. Note that these are in alphabetical order by name, rather than
+            in the order they are used in the circuit.
```

Comment:
```suggestion
        Now, `circuit` has a signature of `(x, y)`. The parameters are ordered alphabetically.
```

Reply:
Looks like we're using markdown instead of reST syntax here:
```suggestion
        Now, ``circuit`` has a signature of ``(x, y)``. The parameters are ordered alphabetically.
```